azext_edge/tests/edge/orchestration/resources/test_assets_unit.py

```diff
@@ -0,0 +1,82 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+
+from typing import Optional
+
+from .conftest import get_base_endpoint, get_mock_resource
+
+ADR_RP = "Microsoft.DeviceRegistry"
+ADR_API_VERSION = "2024-11-01"
+
+
+def get_asset_endpoint(resource_group_name: str, asset_name: Optional[str] = None) -> str:
```

Comment:
seems like quite similar to `get_aep_endpoint` in azext_edge/tests/edge/orchestration/resources/test_aeps_unit.py, except the `resource_path` value difference, should we factor this out to conftest.py?

Reply:
Conftest should minimize additional domain specificity. Also look at the consistency across resources unit test modules, they define endpoints and mock records ... it would be odd for only asset and aeps endpoints to be directly defined in conftest.